README.md

```diff
@@ -108,17 +108,36 @@ If you're using `docker-mailserver` version `v10.1.x` or below, you will need to
 
 ### Get up and running
 
+#### First Things First
+
+**Use `docker-compose up / down`, not `docker-compose start / stop`**. Otherwise, the container is not properly destroyed and you may experience problems during startup because of inconsistent state.
+
+You are able to get a full overview of how the configuration works by either running:
+
+1. `./setup.sh help` which includes the options of `setup.sh`.
+2. `docker run --rm docker.io/mailserver/docker-mailserver:latest setup help` which provides you with all the information on configuration provided "inside" the container itself.
+
+#### Starting for the first time
+
+On first start, you will likely see an error stating that there are no mail accounts and the container will exit. You must now do one of two things:
+
+1. Use `setup.sh` to help you: `./setup.sh email add <user@domain> <password>`. You may need to provide the correct configuration path (to the directory mounted to `/tmp/docker-mailserver` inside the container yourself) with the `-c` option. This will spin up a new container, mount your configuration volume, and create your first account.
```

Comment:
```suggestion
1. Use `setup.sh` to help you: `./setup.sh email add <user@domain> <password>`. You may need  the `-c` option to provide the local path for persisting configuration (_a directory that mounts to `/tmp/docker-mailserver` inside the container_). This will spin up a new container, mount your configuration volume, and create your first account.
```